src/olympia/addons/indexers.py

```diff
@@ -330,8 +330,7 @@ def extract_document(cls, obj):
             # add-ons stored in ES.
             data['platforms'] = [amo.PLATFORM_ALL.id]
             try:
-                data['has_theme_rereview'] = (
-                    obj.persona.rereviewqueuetheme_set.exists())
+                data['has_theme_rereview'] = False
```

Comment:
You can remove that entirely (and the associated checks in `test_indexers.py`)

Reply:
I wasn't sure if it was used externally (though it's not documented, like none of the LWT stuff is).  I'll remove it.

Reply:
https://github.com/mozilla/addons-server/pull/10759/commits/e03f87ba6942fff0d4f09fd1efd639811a4de04d